frameworks/TypeScript/nest/src/bench/bench.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { WorldEntity } from 'models/world.entity';
+import { InjectRepository } from '@nestjs/typeorm';
+
+@Injectable()
+export class BenchService {
+    constructor(
+        @InjectRepository(WorldEntity)
+        private worldRepository: Repository<WorldEntity>){}
+
+    async getOne(){
+
+        return await this.worldRepository.findOne({ where: {id: Math.floor(Math.random() * 10000) + 1}});
+    }
+
+    async getMultiple(totalQueries: string) {
+
+        return new Promise(async (resolve, reject) => {
+            const worldArr = [];
+            const total = this.sanitizeQueries(totalQueries);
+            for (let i = 0; i < total; i++) {
+                worldArr.push(await this.getOne());
+            }
+
+            if (worldArr.length === total) {
+                resolve(worldArr);
+            }
+        });
+    }
+
+    async updateMultiple(totalQueries: string) {
+
+        return new Promise(async (resolve, reject) => {
+            const worldArr = [];
+            const total = this.sanitizeQueries(totalQueries);
+            for (let i = 0; i < total; i++) {
+                let worldToUpdate = await this.getOne();
+                worldToUpdate.randomnumber = Math.floor(Math.random() * 10000) + 1;
+                worldToUpdate = await this.worldRepository.save(worldToUpdate);
+                worldArr.push(worldToUpdate);
+            }
+
+            if (worldArr.length === total) {
```

Comment:
Should you remove this condition? What happens if the total isn't reached? Does the server respond? Does it error with a rejected promise? If you remove the condition and the total is wrong, the toolset will complain about the number of results returned versus the number of results expected.

Reply:
Removed. It was unnecessary.